typescript/examples/vercel-ai/mode/index.ts

```diff
@@ -1,6 +1,7 @@
 import readline from "node:readline";
 
-import { openai } from "@ai-sdk/openai";
+import { createXai } from "@ai-sdk/xai";
```

Comment:
No need to push the example file! :)

Reply:
@0xaguspunk how can I remove it? When it's like already pushed?